pkg/util/iptables/iptables.go

```diff
@@ -328,10 +328,11 @@ func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...st
 
 	// Sadly, iptables has inconsistent quoting rules for comments.
 	// Just unquote any arg that is wrapped in quotes.
-	argsCopy := make([]string, len(args))
-	copy(argsCopy, args)
-	for i := range argsCopy {
-		unquote(&argsCopy[i])
+	var argsCopy []string
+	for i := range args {
+		tmpField := args[i]
+		unquote(&tmpField)
+		argsCopy = append(argsCopy, strings.Fields(tmpField)...)
```

Comment:
Actually just the rules that have quoted multi-word comments get duped..

and this is why I've unquoted and unpacked every word..